src/templates/BillOfLading/BillOfLadingTemplate.tsx

```diff
@@ -6,86 +6,89 @@ import { BillOfLading } from "./types";
-            </div>
-            <div className="w-1/2 border-black border">
-              <div className="p-2">{smallText("Shipped on Board Date (Local Time)")}</div>
+            <div className="flex flex-1">
+              <div className="w-1/2 border-black border">
+                <div className="p-2">{smallText("Declared Value (see clause 7.3)")}</div>
+              </div>
+              <div className="w-1/2 border-black border">
+                <div className="p-2">{smallText("Shipped on Board Date (Local Time)")}</div>
+              </div>
             </div>
           </div>
         </div>
-      </div>
 
-      <div className="w-1/2 border-black border">
-        <div className="p-2">
-          {smallText(
-            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Sagittis id consectetur purus ut faucibus. Diam quam nulla porttitor massa. Eu tincidunt tortor aliquam nulla facilisi cras fermentum. Amet mauris commodo quis imperdiet massa tincidunt. Luctus accumsan tortor posuere ac ut. Eu volutpat odio facilisis mauris sit amet massa vitae tortor. Eros donec ac odio tempor orci dapibus. Varius morbi enim nunc faucibus a pellentesque sit amet. Velit aliquet sagittis id consectetur purus ut. Porta non pulvinar neque laoreet suspendisse interdum consectetur libero. Odio ut sem nulla pharetra diam sit. Nunc sed augue lacus viverra vitae congue eu consequat ac. Eros in cursus turpis massa tincidunt dui ut ornare lectus."
-          )}
-          <div className="text-center mt-4 mb-16">
-            <strong>{smallText("Signed for the Carrier A.P. Moller")}</strong>
-          </div>
-          <hr />
-          <div className="text-center mt-2">
-            <strong>{smallText("As Agent(s) for the Carrier")}</strong>
+        <div className="w-1/2 border-black border">
+          <div className="p-2">
+            {smallText(
+              "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Sagittis id consectetur purus ut faucibus. Diam quam nulla porttitor massa. Eu tincidunt tortor aliquam nulla facilisi cras fermentum. Amet mauris commodo quis imperdiet massa tincidunt. Luctus accumsan tortor posuere ac ut. Eu volutpat odio facilisis mauris sit amet massa vitae tortor. Eros donec ac odio tempor orci dapibus. Varius morbi enim nunc faucibus a pellentesque sit amet. Velit aliquet sagittis id consectetur purus ut. Porta non pulvinar neque laoreet suspendisse interdum consectetur libero. Odio ut sem nulla pharetra diam sit. Nunc sed augue lacus viverra vitae congue eu consequat ac. Eros in cursus turpis massa tincidunt dui ut ornare lectus."
+            )}
+            <div className="text-center mt-4 mb-16">
+              <strong>{smallText(`Signed for the Carrier ${carrierName || ""}`)}</strong>
```

Comment:
is the `|| ""` required?

Reply:
As noted from the acceptance criteria, there seems to have a chance of carrier name being blank thus I add in || "" to prevent a display of undefined if they leave the fields out.